components/Footer.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/no-deprecated */
```

Comment:
why this

Reply:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-deprecated.md